recipes/verta/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "verta" %}
+{% set version = "0.8.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 24db7f5922fd63db8094717a020560435e121f5bf9c7c9478bad5938fe08947e
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python >=3.5
+    - pip
+  run:
+    - python >=3.5
+    - googleapis-common-protos ~=1.5
```

Comment:
What does the `~=` mean? I haven't seen it before and using `>=1.5` is normally what is needed.

Reply:
`~=1.5` is [shorthand for `>=1.5, ==1.*`](https://www.python.org/dev/peps/pep-0440/#compatible-release) so that if breaking changes are released in the dependency's `2.0`, this package would preemptively prohibit the upgrade.

I've been seeing it as a best practice here and there, but if it might cause problems I'd be more than willing to change them to `>=`!

Reply:
`~=` is very nice, thanks for explaining. It seems to work with conda so I see no harm in using it and I will be from now.

One thing to consider is that most Python packages don't have many breaking changes for major version updates and it is (almost) impossible to override these pinnings. For example I expect joblib 1.0 would be fine. I don't feel too strongly either way so it's up to you to decide what is best for each package as both `>=` and `~=` have disadvantages.

In general conda-forge tends favour being relaxed about this and there are methods for fixing it afterwards if there is a serious problem.

Reply:
Hmm hard for me to say as well. I'll have to give this some more thought. Could I ping you once I've come up with a decision?

Reply:
@chrisburr I've decided that, since updating a Conda recipe is a nontrivial process, it's better to give the user the flexibility to upgrade or downgrade dependencies themselves, rather than wait for the PR approval cycle.

If there aren't any more concerns and the checks succeed, I'd be satisfied with these changes. Thank you for your help and time, Chris!